.eslintrc.js

```diff
@@ -34,6 +34,87 @@ module.exports = {
     'ember/no-empty-attrs': 2,
     'ember/no-jquery': 2,
     'ember/no-observers': 2,
+
+    'ember/order-in-models': [
+      2,
+      {
+        order: ['attribute', 'relationship', 'single-line-function', 'multi-line-function'],
```

Comment:
Can we multi-line this to make ordering clearer pls?

Reply:
This is prettier magic ✨

The way to get around this is defining https://eslint.org/docs/rules/array-element-newline. What option do you think we should use?

Reply:
Andy, what do you think would be ideal here?

Reply:
thx for adding this Siva! 🥇

Reply:
@vladucu my thinking was just that because this array is specifically about ordering of things, having each item on its own line would be clearer (and also easier to update). The `consistent` option seems to do allow us to do what we want here, right?

Reply:
yeah, I could go with `consistent`....it's definitely the one that makes more sense
what do you think Siva?